api/app/utils/review.inference.js

```diff
@@ -17,14 +17,11 @@ const buildIsActiveCondition = (currentTime) => {
     createdAt: { [Op.gte]: timeStringAt24HoursPrior },
     mergedAt: { [Op.eq]: null }, // no mergedAt
     archivedAt: { [Op.eq]: null }, // no archivedAt
+    submittedAt: { [Op.eq]: null }, // no submittedAt
   };
 };
-const buildIsPendingCondition = (currentTime) => {
-  const timeStringAt24HoursPrior = new Date(
-    currentTime - unixEpochTimeInMilliseconds
-  ).toISOString();
+const buildIsPendingCondition = (_currentTime) => {
```

Comment:
we ain't using `currentTime` in this function. why keep it?

Reply:
Nothing really, the other similar build functions follow that convention and it did as well up until this refactor. Since the value is not being used I prepended an `_` to it.

Reply:
we should remove it since it serves no purpose here.